config/condor_mapfile.osg

```diff
@@ -1,3 +1,4 @@
+GSI ".*,/[/A-Za-z0-9\.]*/Role=[A-Za-z0-9]*/Capability=NULL[,/A-Za-z0-9\.=]*$" GSS_ASSIST_GRIDMAP
```

Comment:
```suggestion
GSI ".*,/[/A-Za-z0-9\.]*/Role=[A-Za-z0-9\.]*/Capability=NULL$" GSS_ASSIST_GRIDMAP
```

Two small tweaks:
1. I think `.` is valid for Roles.
2. Nothing is permitted past the capability.

Reply:
> Nothing is permitted past the capability.

Except additional VOMS FQANs, which come in comma separated

Reply:
Oh duh, it just matches the last VOMS FQAN and any preceding ones are caught in the `.*`.